From 0c3ee6184361c861706d2e34a81af4018a70e50e Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 26 Jun 2014 14:59:24 +0800
Subject: [PATCH 330/602] ENGR00320078 Change /mnt/shm's permission to align
 with vpu device

VPU will use /mnt/shm for creating share memory for different vpu instance

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index ce5002c..9c3244c 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -100,8 +100,8 @@ loglevel 3
     mount tmpfs tmpfs /mnt/obb mode=0755,gid=1000
 
     # shared memory used by middleware
-    mkdir /mnt/shm  0775 system graphics
-    mount tmpfs tmpfs /mnt/shm mode=0775,uid=1000,gid=1003,size=1m
+    mkdir /mnt/shm  0775 media system
+    mount tmpfs tmpfs /mnt/shm mode=0775,uid=1013,gid=1000,size=1m
     write /proc/sys/kernel/panic_on_oops 1
     write /proc/sys/kernel/hung_task_timeout_secs 0
     write /proc/cpu/alignment 4
-- 
1.8.0

